source/reference/cutover-process.txt

```diff
@@ -86,3 +101,13 @@ Procedure
       When the ``mongosync`` progress response indicates that the 
       ``mongosync`` state is ``COMMITTED``, the cutover process is
       complete. 
+
+Learn More
+----------
+
+* :ref:`c2c-verification`
```

Comment:
Not our standard. Use - characters, not *.

Reply:
Fixed